modifiers/ReplaceBlockPairs.groovy

```diff
@@ -2,16 +2,15 @@ import net.querz.mcaselector.io.mca.ChunkData
 import net.querz.nbt.CompoundTag
 
 /**
- * Defines pairs of blocks to replace.
+ * Specify blocks as comma-separated pairs in the format 'source:original_block': 'source:new_block'
```

Comment:
switch this line with the line below. This way the first line describes what the script does and the second how to configure it.